lib/fathead/mdn_css/parse.pl

```diff
@@ -207,6 +207,7 @@
         }
 
         $code = trim($code);
+        $code =~ s!\\0!\\\\0!;
```

Comment:
@hchienjo it looks like there's more to the issue here.

For https://developer.mozilla.org/en-US/docs/Web/CSS/content on the 6th line of the code snippet, it says `/* e.g. \00A0 for &nbsp; */` -- If you look at the output.txt we have `/* e.g. \00A0 for */` -- we're converting the `&nbsp` to the non-breaking space character.

Can you ensure this doesn't happen? I assume this is a result of the way we're getting the raw text from the DOM?